packages/gatsby-transformer-sqip/src/extend-node-type.js

```diff
@@ -42,7 +41,12 @@ module.exports = async args => {
   return {}
 }
 
-async function sqipSharp({ type, cache, getNodeAndSavePathDependency }) {
+async function sqipSharp({ type, cache, getNodeAndSavePathDependency, store }) {
+  const program = store.getState().program
+  const cacheDir = resolve(`${program.directory}/.cache/sqip/`)
```

Comment:
Hint: This changes the cacheDir of SQIP to be aligned with the cached Contentful assets. Both of these do not need to be publicly available.